Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -94,6 +94,25 @@
     from System.Runtime.CompilerServices import ConditionalWeakTable
     IPY_SEEN_MODULES = ConditionalWeakTable[object, object]()
 
+    clr.AddReference('Microsoft.Dynamic')
```

Comment:
I copied all of these from the repl implementation, and I didn't try to minimize them to see which ones were really needed.  Without any of it, executing code in modules was failing.